src/test/compile-fail/rfc1623.rs

```diff
@@ -7,7 +7,7 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
-
```

Comment:
Nit: can you make another test that specifically uses the feature w/o the feature gate and check that it reports an error